cmd/tsconnect/package.json

```diff
@@ -11,7 +11,8 @@
     "tailwindcss": "^3.1.6",
     "typescript": "^4.7.4",
     "xterm": "mihaip/xterm.js#95100e3c870b59348bb4fa5504bab9f9317bac67",
-    "xterm-addon-fit": "^0.5.0"
+    "xterm-addon-fit": "^0.5.0",
+    "xterm-addon-web-links": "0.7.0-beta.6"
```

Comment:
Had to use the beta version of the addon because we're using the HEAD version of xterm.js, which has some API changes that the addon needs to pick up (xtermjs/xterm.js@7a17a26e18c9640821508cb987ddf43bb505c6dc).